src/textbooks/empty-placeholder/EmptyPlaceholder.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
[nit]: Please remove this unnecessary React import

Reply:
Fixed